src/components/daily/foodtooltip/FoodTooltip.js

```diff
@@ -29,18 +29,45 @@ class FoodTooltip extends React.Component {
     return _.get(food, 'nutrition.carbohydrate.net', 0);
   }
 
+  getPrescribedCarbs(food) {
+    return _.get(food, 'prescribedNutrition.carbohydrate.net', 0);
+  }
+
+  isPrescribed(food) {
+    const prescriptor = _.get(food, 'prescriptor');
+    if (prescriptor !== undefined) {
```

Comment:
I would have been rather more specific or less:
```suggestion
    if (prescriptor) {
```

or
```suggestion
    if (typeof prescriptor === 'string') {
```